engine/src/main/java/org/pentaho/di/repository/KettleRepositoryNotFoundException.java

```diff
@@ -0,0 +1,59 @@
+/*! ******************************************************************************
+ *
+ *
+ * Pentaho Data Integration
+ * Copyright (C) 2019 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+package org.pentaho.di.repository;
+
+import org.pentaho.di.core.exception.KettleException;
+
+/**
+ * This exception is thrown when there was an authentication error, while connecting into a repository.
```

Comment:
Description does not match exception name. Copy paste error?

Reply:
Updated! *facepalm*